content/docs/instrumenting/exposition_formats.md

```diff
@@ -5,104 +5,96 @@ sort_rank: 6
 `summary`, or `untyped`, defining the type for the metric of that name. Only
-one `TYPE` line may exist for the same metric name. The `TYPE` line for a
-metric name has to appear before the first sample is reported for that metric
+one `TYPE` line may exist for a given metric name. The `TYPE` line for a
+metric name must appear before the first sample is reported for that metric
 name. If there is no `TYPE` line for a metric name, the type is set to
 `untyped`. Remaining lines describe samples, one per line, with the following
-syntax (EBNF):
+syntax ([EBNF](https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form)):
+
+```
+metric_name [
+  "{" label_name "=" `"` label_value `"` { "," label_name "=" `"` label_value `"` } [ "," ] "}"
+] value [ timestamp ]
+```
 
-    metric_name [
-      "{" label_name "=" `"` label_value `"` { "," label_name "=" `"` label_value `"` } [ "," ] "}"
-    ] value [ timestamp ]
+`metric_name` and `label_name` have the usual Prometheus expression language restrictions:
 
-`metric_name` and `label_name` have the usual Prometheus expression language restrictions. `label_value` can be any sequence of UTF-8 characters, but the backslash, the double-quote, and the line-feed characters have to be escaped as `\\`, `\"`, and `\n`, respectively.
-`value` is a float, and timestamp an `int64` (milliseconds since epoch, i.e. 1970-01-01 00:00:00 UTC, excluding leap seconds), represented as required by the [Go strconv package](http://golang.org/pkg/strconv/) (see functions `ParseInt` and `ParseFloat`). In particular, `Nan`, `+Inf`, and `-Inf` are valid values.
+* `label_value` can be any sequence of UTF-8 characters, but the backslash, the double-quote, and the line-feed characters have to be escaped as `\\`, `\"`, and `\n`, respectively.
+* `value` is a float
+* the `timestamp` is an `int64` (milliseconds since epoch, i.e. 1970-01-01 00:00:00 UTC, excluding leap seconds), represented as required by the [Go strconv package](http://golang.org/pkg/strconv/) (see functions [`ParseInt`](https://golang.org/pkg/strconv/#ParseInt) and [`ParseFloat`](https://golang.org/pkg/strconv/#ParseFloat)). In addition to standard integers, `Nan`, `+Inf`, and `-Inf` are valid values representing not a number, positive infinity, and negative infinity, respectively.
 
-All lines for a given metric must be provided as one uninterrupted group, with
+#### Grouping and sorting
+
+All lines for a given metric must be provided as one single group, with
 the optional `HELP` and `TYPE` lines first (in no particular order). Beyond
 that, reproducible sorting in repeated expositions is preferred but not
 required, i.e. do not sort if the computational cost is prohibitive.
 
-Each line must have a unique combination of metric name and labels. Otherwise,
+Each line must have a unique combination of a metric name and labels. Otherwise,
 the ingestion behavior is undefined.
 
+#### Histograms and summaries
```

Comment:
When introducing these new sub-headers, we'll then have to introduce them for all sub-sections of the text format, including the example section below (the example is no longer about histograms and summaries in particular, but about the text format in general).

Reply:
Yep, you're right. Missed this one.